src/main/java/org/shipkit/auto/version/VersionConfig.java

```diff
@@ -99,6 +99,10 @@ static boolean isSupportedVersion(String version) {
         return version.matches("\\d+\\.\\d+\\.\\d+");
     }
 
+    static boolean isSnapshot(String tag, String tagPrefix) {
```

Comment:
Let's add javadoc

Reply:
@mockitoguy Thanks for your review! As you suggested I refactored isSupportedVersion() and isSnapshot() and covered these methods with unit tests. I also added javadoc.